### ISSUE TITLE
v 3.3 downgrades symfony/workflow to 5.4.3

### ISSUE THREAD

**Issue by klimenttoshkov** [zerodahero/laravel-workflow#55]
```
Using version ^3.3 for zerodahero/laravel-workflow
./composer.json has been updated
Running composer update zerodahero/laravel-workflow
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - zerodahero/laravel-workflow[v3.3.0, ..., 3.x-dev] require symfony/event-dispatcher ^5.0 -> found symfony/event-dispatcher[v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.3 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - zerodahero/laravel-workflow[v3.3.2, ..., v3.3.3] require symfony/event-dispatcher-contracts ^2.4 -> found symfony/event-dispatcher-contracts[v2.4.0, ..., 2.5.x-dev] but the package is fixed to v3.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires zerodahero/laravel-workflow ^3.3 -> satisfiable by zerodahero/laravel-workflow[v3.3.0, ..., 3.x-dev].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require zerodahero/laravel-workflow:*" to figure out if any version is installable, or "composer require zerodahero/laravel-workflow:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
sag@Sag-MacBook-Pro-15 zz_api % composer require zerodahero/laravel-workflow -W
Using version ^3.3 for zerodahero/laravel-workflow
./composer.json has been updated
Running composer update zerodahero/laravel-workflow --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Lock file operations: 2 installs, 2 updates, 0 removals
  - Upgrading klimenttoshkov/laravel-currency (dev-master 7a55352 => dev-master fd7c21c)
  - Downgrading symfony/event-dispatcher-contracts (v3.0.0 => v2.5.0)
  - Locking symfony/workflow (v5.4.3)
  - Locking zerodahero/laravel-workflow (v3.3.3)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 2 installs, 2 updates, 0 removals
  - Downloading symfony/event-dispatcher-contracts (v2.5.0)
  - Downloading symfony/workflow (v5.4.3)
  - Downloading zerodahero/laravel-workflow (v3.3.3)
  - Downgrading symfony/event-dispatcher-contracts (v3.0.0 => v2.5.0): Extracting archive
  - Upgrading klimenttoshkov/laravel-currency (dev-master 7a55352 => dev-master fd7c21c): Source already present
  - Installing symfony/workflow (v5.4.3): Extracting archive
  - Installing zerodahero/laravel-workflow (v3.3.3): Extracting archive
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Generating optimized autoload files
```

**Comment by zerodahero** [comment]
Looks like we're due for an upgrade on the underlying Symfony packages. I'll take a look at any breaking changes and push out a new version for that soon.

**Comment by klimenttoshkov** [comment]
More extras coming in:

```
    - zerodahero/laravel-workflow 3.4.x-dev requires illuminate/contracts ^7.1|^8.0 -> satisfiable by illuminate/contracts[v7.1.0, ..., v7.30.6, v8.0.0, ..., v8.83.0].
    - Only one of these can be installed: illuminate/contracts[v5.6.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.0, v9.0.0], laravel/framework[v9.0.0]. laravel/framework replaces illuminate/contracts and thus cannot coexist with it.
```

**Comment by zerodahero** [comment]
This will be addressed in v4.0, to be released soon.

**Comment by zerodahero** [comment]
This was addressed in [v4.0.0](https://github.com/zerodahero/laravel-workflow/releases/tag/v4.0.0), please let me know if there's still any issues remaining for this.
